src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,47 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+// eslint-disable-next-line max-len, object-curly-newline
+export const MovieCard: React.FC<Movie> = ({ title, description, imgUrl, imdbUrl, imdbId }) => (
```

Comment:
The MovieCard component should receive a single prop of type Movie instead of destructuring the Movie object.